ansible/files/restart-as-needed.sh

```diff
@@ -0,0 +1,34 @@
+#! /usr/bin/env bash
```

Comment:
I can't say that I'm a friend of the `/usr/bin/env` hack, but well...

(It's not as portable as it claims to be: POSIX doesn't allow any arguments for the command.)

Reply:
@ulm it's not as flexible as one could wish, but let's take that discussion offline…

Reply:
@ulm thanks for the review, merging…